### DISCUSSION TITLE
is it possible to add fields to log by LAL or some ways?

### DISCUSSION THREAD

**Discussion by chongchaoyu** [apache/skywalking#9242] [Q&A] [answered]
I want to add my customed fields to log, and filter logs on kibana, is it possible?
![94DCC6DE8762838BA0E1CB9ABECB098A](https://user-images.githubusercontent.com/12747983/174070676-f96e96a5-cc23-4f37-a264-4e917c2b7b1d.jpg)

**Comment by wu-sheng** [comment, marked as answer]
From a configuration perspective, no, there isn't a way to do so.

**Reply by chongchaoyu** [reply]
Thank you, can a simple plugin reach this? If it is possible, could you give some advice about it?

**Reply by wu-sheng** [reply]
I am afraid not. Visualizing from Kibana is not considered by the community for NOW.

**Reply by chongchaoyu** [reply]
Got it, thanks

**Reply by MQPearth** [reply]
@wu-sheng 
Hope to provide some ways to implement this function. Since Skywalking already provides the functionality to store logs in Elasticsearch, I don't need to use tools like Filebeat to synchronize logs anymore.

**Reply by wu-sheng** [reply]
Don't have such plan. Storage is pluggable, what you asked doesn't make sense to us.
